src/services/FileSystemService.ts

```diff
@@ -76,6 +76,14 @@ class FileSystemService {
     const items = await RNFS.readDir(this.getTemporaryDir());
 
     items.forEach(async (item) => {
+      // Some library is writing this file
+      // in the tmp directory, on startup
+      // something is trying to pick the file,
+      // so we avoid deleting this file. Otherwise
+      // the app crashes on startup.
+      if (item.path.includes('NSIRD')) {
```

Comment:
which library is doing this?

Reply:
I suspect is the OS itself, I provided a link with useful info in the PR